test-float-features.lisp

```diff
@@ -0,0 +1,107 @@
+(in-package #:cl-user)
+
+(defpackage #:float-features/tests
+  (:nicknames #:org.shirakumo.float-features/tests)
+  (:use #:cl))
+
+;;; to run (parachute:test :float-features/tests)
+(in-package #:float-features/tests)
+
+(parachute:define-test float-features-1
+    :compile-at :execute
```

Comment:
Is there a good reason these tests need to be compiled at runtime?

Reply:
yes constant flolding by sbcl

Reply:
removed